File: backend/src/main/java/org/jdk/project/dto/digitaltwin/DigitalTwinRegionDto.java
package org.jdk.project.dto.digitaltwin;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class DigitalTwinRegionDto {
  private Long id;
  private String regionName;
  private String areaName;
  private String roomCode;
  private String deptName;
  private String coordinate;
  private Integer maxCapacity;
  private Integer curCapacity;
}

File: backend/src/main/kotlin/org/jdk/project/dto/digitaltwin/DigitalTwinRegionDto.kt
package org.jdk.project.dto.digitaltwin

data class DigitalTwinRegionDto(
    var id: Long? = null,
    var regionName: String? = null,
    var areaName: String? = null,
    var roomCode: String? = null,
    var deptName: String? = null,
    var coordinate: String? = null,
    var maxCapacity: Int? = null,
    var curCapacity: Int? = null,
)
